### DISCUSSION TITLE
Feature Request: Change colour in each frame at once

### DISCUSSION THREAD

**Discussion by ArtistBirdy** [keiwando/toonsquid#306] [Ideas]
It would be so helpful if this feature comes in toon squid 👇🏻in next update 
[YouTube video Toon boom feature](https://youtu.be/_O7wsI3H4_8) 
 Thanks in advance 🙏

**Comment by DWrightAnimation** [comment]
This would be very useful, but how would it work with characters with multiple colors?

**Reply by ArtistBirdy** [reply]
it woud be helpful for specific characters and props that will in same colour in animation, also if we can use many colour pallets for the different objects per animation project, also rename them make colours more organised. like if we change one colour from any of the object, then it'll be affected to same object in every frame at same time.
